libcontainer/cgroups/fs/blkio.go

```diff
@@ -226,12 +230,15 @@ func getStats(path string, stats *cgroups.Stats) error {
 	if blkioStats, err = getBlkioStat(filepath.Join(path, "blkio.throttle.io_service_bytes")); err != nil {
 		return err
 	}
-	stats.BlkioStats.IoServiceBytesRecursive = blkioStats
+	//	stats.BlkioStats.IoServiceBytesRecursive = blkioStats
```

Comment:
Actually, while testing my code I realized testcase [``TestNonCFQBlkioStats``](https://github.com/docker/libcontainer/pull/167/files#diff-f734435c91f3ff99263df9f4577a3e15R208) doesn’t executes for non-CFQ condition as ``blkio.io_serviced_recursive`` attribute is present. As result, It always executes CFQ case only.

If some tools depends on ``IoServiceBytesRecursive`` and do not honour ``ThrottleIoServiceBytes`` , I can re-test the results with them.